docs/references/architecture/adr-114-undo-109.md

```diff
@@ -92,11 +92,11 @@ Column legend:
 | bits                       | keep private       | 0                                      | 26             | low        | [bits-url]        |
 | blocksync                  | keep private       | 0                                      | 6              | low        | [blocksync-url]   |
 | clist                      | keep private       | 0                                      | 24             | low        | [clist-url]       |
+| indexer                    | keep private       | 0                                      | 0              | low        | [indexer-url]     |
 | net                        | keep private       | 1                                      | 40             | low        | [net-url]         |
 | statesync                  | **🧹 make public** | 1                                      | 16             | medium     | [statesync-url]   |
-| evidence                   | **🧹 make public** | 1                                      | 26             | high       | [evidence-url]    |
-| consensus                  | **🧹 make public** | 1                                      | 64             | high       | [consensus-url]   |
-| indexer                    | **🧹 make public** | 2                                      | 100            | medium     | [indexer-url]     |
+| evidence                   | keep private       | 1                                      | 26             | high       | [evidence-url]    |
+| consensus                  | keep private       | 1                                      | 64             | high       | [consensus-url]   |
```

Comment:
See the remark below in this document

> For `evidence` and `consensus`: There is a single project we have identified using APIs from these modules,
specifically <https://github.com/forbole/juno>. The maintainers of this project have agreed it is not a problem
for them if we keep the two modules private.